### PR TITLE
Bunch of stuff

### PR DESCRIPTION
**Pull request by kymbibop**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "dev": "vite",
+    "dev": "vite --host",
```
Comment (dannyforest) [inline package.json]: this doesn't belong in this branch!

File: src/assets/main.css
Comment (dannyforest) [inline src/assets/main.css:1]: dangerous.

File: src/components/AddToList.vue [hunk 2]
```diff
@@ -1,11 +1,47 @@
...
+}
 
 </script>
 
 <template>
-
+    <form id="addToList" @submit.prevent="handleSubmit">
+        <h2>Add to the list</h2>
+        <input v-model="name" type="text" placeholder="Your name" required></input>
+        <input v-model="giftsList" type="text" placeholder="Add your gift name" required></input>
```
Comment (dannyforest) [inline src/components/AddToList.vue]: We should be able to add multiple gifts if giftsList is an array. Let's do in next sprint.

File: src/components/AddToList.vue [hunk 3]
```diff
@@ -1,11 +1,47 @@
 <script setup>
+import { ref } from 'vue';
+const emit = defineEmits(['add']);
+const name = ref('');
+const giftsList = ref([]);
```
Comment (dannyforest) [inline src/components/AddToList.vue]: Could be rename to either giftList or gifts

File: src/components/AddToList.vue [hunk 4]
```diff
@@ -1,11 +1,47 @@
...
+const naughty = ref(false);
+
+const handleSubmit = () => {
+    emit('add',addname.value, giftsList.value, naughty.value);
```
Comment (dannyforest) [inline src/components/AddToList.vue]: addname is not defined

File: src/components/__test__/AddToList.spec.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+import AddToList from '@/components/AddToList.vue';
+
+
+describe('AddToList.vue', () => {
+    let wrapper;
+
+    beforeEach(() => {
+        wrapper = mount(AddTolist);
+    });
+    it('emits the form on submit', async () => {
```
Comment (dannyforest) [inline src/components/__test__/AddToList.spec.js:13]: tests does not pass!

File: src/components/icons/IconCommunity.vue
Comment (dannyforest) [inline src/components/icons/IconCommunity.vue:1]: should not be part of this feature

File: src/components/icons/IconDocumentation.vue
Comment (dannyforest) [inline src/components/icons/IconDocumentation.vue:1]: should not be part of this feature

File: src/components/icons/IconEcosystem.vue
Comment (dannyforest) [inline src/components/icons/IconEcosystem.vue:1]: should not be part of this feature

File: src/components/icons/IconSupport.vue
Comment (dannyforest) [inline src/components/icons/IconSupport.vue:1]: should not be part of this feature

File: src/components/icons/IconTooling.vue
Comment (dannyforest) [inline src/components/icons/IconTooling.vue:1]: should not be part of this feature

File: src/views/HomeView.vue
Comment (dannyforest) [inline src/views/HomeView.vue:1]: This shouldn't change!! 😡

**Review by dannyforest — CHANGES_REQUESTED** [review]
This is garbage. try again

**Comment by kymbibop** [comment]
> This is garbage. try again

Very constructive, thanks?

**Review by dannyforest — APPROVED** [review]

**Pull request merged**